### PR TITLE
Sw-6661 feedback page

### PR DESCRIPTION
**Pull request by kate-49**

### CODE REVIEW

File: internal/server/feedback-form.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			if verr, ok := err.(sirius.ValidationError); ok {
+				vars.Errors = verr.Errors
+				vars.Error = verr.Message
+				vars.Form = feedbackForm
+				return tmpl.ExecuteTemplate(w, "page", vars)
+			} else if err != nil {
+				return err
+			} else {
+				vars.Success = true
+			}
```
Comment (Nicholassully) [inline internal/server/feedback-form.go:50]: Could this block be a bit clearer like 
if verr, ok := err.(sirius.ValidationError); ok {
    vars.Errors = verr.Errors
    vars.Error = verr.Message
    vars.Form = feedbackForm
    return tmpl.ExecuteTemplate(w, "page", vars)
} else if err != nil {
    return err
}

vars.Success = true

File: internal/sirius/add_feedback.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	var body bytes.Buffer
+	var err error
+
+	if len(form.Message) == 0 {
+		return ValidationError{
+			Message: "isEmpty",
+		}
+	}
+
+	if len(form.Message) > 900 {
```
Comment (Nicholassully) [inline internal/sirius/add_feedback.go:20]: Maybe have this len(form.Message) as a var and that means you only have to look at it once?

File: internal/sirius/add_feedback.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		return ValidationError{
+			Message: "stringLengthTooLong",
```
Comment (Nicholassully) [inline internal/sirius/add_feedback.go:22]: Is there a test for this validation error?

File: internal/sirius/client.go [hunk 4]
```diff
@@ -85,3 +89,11 @@ func (c *Client) newRequest(ctx Context, method, path string, body io.Reader) (*
 
 	return req, err
 }
+
+func getContext(cookies []*http.Cookie) Context {
+	return Context{
+		Context:   context.Background(),
+		Cookies:   cookies,
+		XSRFToken: "abcde",
```
Comment (Nicholassully) [inline internal/sirius/client.go:97]: Is this XSRToken correct? It look strange

File: internal/server/feedback-form.go [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+	Path    string
+	Success bool
+	Errors  sirius.ValidationErrors
+	Error   string
+	Form    model.FeedbackForm
+}
+
+func feedbackForm(client FeedbackFormClient, tmpl Template) Handler {
+	return func(perm sirius.PermissionSet, w http.ResponseWriter, r *http.Request) error {
+		if r.Method != http.MethodGet && r.Method != http.MethodPost {
```
[the remark this answers is not in the document]
Comment (kate-49) [reply internal/server/feedback-form.go:23]: It returns a status error of not allowed if it's not a Get OR a post

File: internal/sirius/add_feedback.go (hunk 3 above)
Comment (kate-49) [reply internal/sirius/add_feedback.go:22]: Yup TestAddFeedbackStringTooLongValidationError in add_feedback_test.go

File: internal/sirius/add_feedback.go (hunk 2 above)
Comment (kate-49) [reply internal/sirius/add_feedback.go:20]: I'm not sure I follow, if it was a variable I'd only use this once?

File: internal/sirius/client.go (hunk 4 above)
Comment (kate-49) [reply internal/sirius/client.go:97]: You're right, not sure what I did there

**Review by Nicholassully — APPROVED** [review]

**Pull request merged**